### ISSUE TITLE
Test vs Inference results for COVID sensitivity

### ISSUE THREAD

**Issue by AlexTS1980** [haydengunraj/COVIDNet-CT#8]
When I evaluate Model A on the test split (21191 imgs) with the `test `method in the script, I get roughly the results in the paper (97.44% for COVID sensitivity), but when I run the `inference `on the same split, the results are noticeably worse (94.47%). What could cause it? I'm not very familiar with tensorflow, perhaps there's a glitch somewhere.

**Comment by Byshkin** [comment]
I have similar problem. If the model is trained from scratch the validation accuracy does not go above 95%

**Comment by AlexTS1980** [comment]
> I have similar problem. If the model is trained from scratch the validation accuracy does not go above 95%

I  trained my model on 3K images from COVIDx-CT split (1K/class) without any data/class augmentations, got very good results on the full test split (https://github.com/AlexTS1980/COVID-CT-Mask-Net). Can you train their model on this small split, see what you get?

**Comment by haydengunraj** [comment]
@AlexTS1980 with respect to the original inference issue, I'll take a look at the code and see if I can find the problem (thanks for bringing this to my attention!).

@Byshkin for training from scratch, you may need to play around with different optimizers, LRs, LR schedulers, etc. The fine-tuning code we provide is intended as a reference for simple fine-tuning, but achieving good performance from scratch may require more sophisticated changes. Also note that the checkpoints we provide here were pretrained on ImageNet, so training directly on COVIDx-CT from scratch may not achieve the same performance.

**Comment by haydengunraj** [comment]
@AlexTS1980 does the resolution in #9 (setting `auto_crop=True`) solve the inference issue?
